src/adapters/expressController.ts

```diff
@@ -92,34 +107,32 @@ export default class ExpressH5PController {
 
     public getLibraryFile = async (
         req: express.Request,
-        res: express.Result
+        res: express.Response
     ): Promise<void> => {
-        const stream = await this.h5pEditor.getLibraryFileStream(
-            LibraryName.fromUberName(req.params.uberName),
-            req.params.file
+        const lib = LibraryName.fromUberName(req.params.uberName);
+        const [stats, stream] = await Promise.all([
+            this.h5pEditor.libraryManager.getFileStats(lib, req.params.file),
+            this.h5pEditor.getLibraryFileStream(lib, req.params.file)
+        ]);
+
+        this.pipeStreamToResponse(
+            path.basename(req.params.file),
```

Comment:
I think you need to pass in the mimetype of the file, but you pass in the filename. I receive warnings in the browser's dev console because of this and some stylesheets are not rendered correctly. 

We already use the npm package mime-types, so you can use it to determine the mimetypes.

Reply:
i see. fixed that.